### ISSUE TITLE
[Adaptive Cards iOS - FoodOrder]: 'How would you like your steak prepared?' label is not associated with its Radio button options.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#6971]
https://microsoft.visualstudio.com/OS/_workitems/edit/33779637
